### PR TITLE
Added Phony Rails to validate and normalize phone numbers

### PR DESCRIPTION
**Pull request by mayankmadan**

### CODE REVIEW

File: app/models/phone.rb [hunk 1]
```diff
@@ -3,6 +3,10 @@ class Phone < ApplicationRecord
   
   validates :number, presence: true, length: {minimum: 9}
 
+  phony_normalize :number, default_country_code: 'IN'
+
+  validates :number, phony_plausible: true
```
Comment (virmani) [inline app/models/phone.rb:8]: what does phony_plausible do?

File: app/models/phone.rb [hunk 2]
```diff
@@ -3,6 +3,10 @@ class Phone < ApplicationRecord
   
   validates :number, presence: true, length: {minimum: 9}
 
+  phony_normalize :number, default_country_code: 'IN'
```
Comment (virmani) [inline app/models/phone.rb:6]: What happens if we have an international patient with a non-Indian number? Not a P0 feature but I want to think this through regardless.

**Review by virmani — COMMENTED** [review]
Left a couple of questions for you.

File: app/models/phone.rb (hunk 1 above)
Comment (mayankmadan) [reply app/models/phone.rb:8]: it validates the phone number.

**Pull request merged**